src/pages/vaults/VaultPositionsTable.tsx

```diff
@@ -47,27 +47,41 @@ export const VaultPositionsTable = ({ className }: { className?: string }) => {
       [
         {
           columnKey: 'market',
-          getCellValue: (row) => row.marketId, // todo lookup asset
+          getCellValue: (row) => row.marketId,
           label: stringGetter({ key: STRING_KEYS.MARKET }),
-          renderCell: ({ marketId, currentLeverageMultiple, currentPosition }) => {
+          renderCell: ({ marketId, currentLeverageMultiple }) => {
             const asset = marketId != null ? marketIdToAssetMetadataMap[marketId] : undefined;
             return (
-              <TableCell stacked slotLeft={<AssetIcon symbol={asset?.id} tw="h-[2.5em]" />}>
-                {asset?.name}
-                <div tw="row gap-0.25">
-                  <$OutputSigned
-                    value={
-                      (currentPosition?.usdc ?? 0) < 0
-                        ? stringGetter({ key: STRING_KEYS.SHORT_POSITION_SHORT })
-                        : stringGetter({ key: STRING_KEYS.LONG_POSITION_SHORT })
-                    }
-                    sign={getNumberSign(currentPosition?.usdc ?? 0)}
-                    type={OutputType.Text}
-                  />
-                  @
-                  <Output type={OutputType.Multiple} value={currentLeverageMultiple} />
-                </div>
-              </TableCell>
+              // eslint-disable-next-line jsx-a11y/no-static-element-interactions
```

Comment:
[suggestion] add `role=button` to silence jsx-a11y lint error (unless there's some reason we can't? in which case comment)